mayastor/src/core/env.rs

```diff
@@ -259,8 +261,8 @@ impl Default for MayastorEnvironment {
     fn default() -> Self {
         Self {
             node_name: "mayastor-node".into(),
-            mbus_endpoint: None,
             grpc_endpoint: None,
+            registration_endpoint: Uri::try_from("https://core:50051").unwrap(),
```

Comment:
Using something like [this](https://github.com/openebs/mayastor/blob/develop/mayastor/src/grpc/mod.rs#L112) would be better IMHO.

Reply:
Done.